projector-launcher/build.gradle.kts

```
@@ -65,9 +67,32 @@ dependencies {
 
 version = "1.1.0"  // npm doesn't support versions like "1.0-SNAPSHOT"
 
-val npmCommand = when (DefaultNativePlatform.getCurrentOperatingSystem().isWindows) {
-  true -> listOf("cmd.exe", "/C", "npm.cmd")
-  false -> listOf("npm")
+val kotlinNodeJsSetupTask = rootProject.tasks.getByName<NodeJsSetupTask>("kotlinNodeJsSetup")
+
+val nodeCommand = if (isFamily(Os.FAMILY_WINDOWS)) {
+  kotlinNodeJsSetupTask.destination
+    .resolve("node.exe")
+}
+else {
+  kotlinNodeJsSetupTask.destination
+    .resolve("bin")
+    .resolve("node")
+}
```

Speaking of refactoring and making code shorter in terms of removing duplicated or unneeded code, I would suggest the following changes:

```suggestion
val nodeDir get() = kotlinNodeJsSetupTask.destination

val nodeCommand = when (isFamily(Os.FAMILY_WINDOWS)) {
  true -> nodeDir.resolve("node.exe")
  false -> nodeDir.resolve("bin/node")
}
```

Also, it replaces `if` with `when` as it was before (and it's better to avoid changing such stuff if it's already written and there is no clear advantage of changing it).